_posts/2021/2021-04-26-order-by-created-and-updated-with-a-concern.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Order by created_at and updated_at with a concern"
+description: "If you sort models by timestamps regularly, this is nice"
+layout: article
+category: ruby
+image:
+  base: "2021/order-by-created-and-updated-with-a-concern"
+  alt: "Food Truck Festival ’ drive-by’"
+  source: "https://unsplash.com/photos/1SjD5ZEiUsA"
+  credit: "Micheile Henderson"
+---
+
+Rails provides `scopes` as a way to encapsulate your regularly used queries. I've suggested before that it's a good idea to [keep calls to the query API within models](/ruby/only-use-named-scopes-ouside-models/).
+
+Rails also includes [Active Support's concerns](https://api.rubyonrails.org/classes/ActiveSupport/Concern.html) as a way to group functionality that's used in multiple places. It's a wrapper around a standard Ruby `module`.
+
+We're often sorting models in our applications by the time they are created or updated, using the [built-in timestamps](https://api.rubyonrails.org/classes/ActiveRecord/Timestamp.html).
```

Comment:
Maybe tie this paragraph to the last one by saying something like: something we often want to do in multiple places in our app is sort models based on...

Will help the flow a little bit.